xds/internal/balancer/edsbalancer/eds_test.go

```diff
@@ -109,6 +109,8 @@ func (f *fakeEDSBalancer) handleEDSResponse(edsResp xdsclient.EndpointsUpdate) {
 
 func (f *fakeEDSBalancer) updateState(priority priorityType, s balancer.State) {}
 
+func (f *fakeEDSBalancer) updateServiceRequestsCounter(serviceName string) {}
```

Comment:
Add a channel to verify `serviceName` is expected.

Reply:
Added the channel. Where should that be verified? I'm guessing added to one of the waitFor functions below?